src/annmas/segment/command.py

```diff
@@ -85,21 +86,30 @@ def main(threads, output_bam, do_simple_splitting, keep_delimiters, input_bam):
     # Start worker sub-processes:
     worker_process_pool = []
     for _ in range(threads):
-        p = mp.Process(target=_sub_process_work_fn, args=(process_input_data_queue, results))
+        p = mp.Process(
+            target=_sub_process_work_fn, args=(process_input_data_queue, results)
+        )
         p.start()
         worker_process_pool.append(p)
 
     pysam.set_verbosity(0)  # silence message about the .bai file not being found
     with pysam.AlignmentFile(
         input_bam, "rb", check_sq=False, require_index=False
-    ) as bam_file, tqdm.tqdm(desc="Progress", unit=" read", colour="green", file=sys.stdout) as pbar:
+    ) as bam_file, tqdm.tqdm(
+        desc="Progress",
+        unit=" read",
+        colour="green",
+        file=sys.stderr,
+        leave=False,
+        disable=input_bam == "-",
```

Comment:
When data is piped in from `annotate`, you get two progress meters clobbering each other.  I disable this progress meter in that scenario.  Again, not _entirely_ sure this is a great idea.

Reply:
I don't know that there's a good way to solve this - because the processes are completely independent, the two tqdm instances will just naturally interfere with each other since they have no way of knowing the other instance exists.

I think this may be as good a solution as we can get (barring some hacking [of the console cursor](https://stackoverflow.com/a/23121189) or similar)

Reply:
Actually, in this case we may want to revert to a logged progress statement rather than the bar itself.  That way we can still have progress updates, but without the bars getting mangled.